src/Microsoft.AspNetCore.Authentication.Cookies/CookieAuthenticationOptions.cs

```diff
@@ -23,6 +22,7 @@ public CookieAuthenticationOptions()
             ReturnUrlParameter = CookieAuthenticationDefaults.ReturnUrlParameter;
             ExpireTimeSpan = TimeSpan.FromDays(14);
             SlidingExpiration = true;
+            CookieSameSite = SameSiteEnforcementMode.None;
```

Comment:
Probably. Strict would break, for example, links to a protected resource from another site. But given we try to be secure by default ... yea, strict.